cmd/promtool/main.go

```diff
@@ -191,6 +191,13 @@ func checkFileExists(fn string) error {
 	return err
 }
 
+func addScheme(urlString string) (string, error) {
+	// Add HTTP Scheme to urlString
+	urlObject, err := url.Parse(urlString)
+	urlObject.Scheme = "http"
```

Comment:
This would lose a https scheme. It's generally easiest to check if the string already has http/https

Reply:
Thanks for the feedback. I'll modify the function to add the scheme only if one doesn't already exist.

Reply:
This has been done in 5300e2a